src/pages/blog.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import Link from 'gatsby-link'
+import styled from 'react-emotion'
+
+const BlogContainer = styled('div')({
+  display: 'flex',
+  'flex-wrap': 'wrap',
+  '-webkit-box-pack': 'start',
+  'justify-content': 'center',
+  background: '#eee',
+})
+
+const BlogEntry = styled('a')({
+  color: 'inherit',
+  width: '50%',
+  'max-width': '100%',
+  display: 'flex',
+  'text-decoration': 'none',
+})
+
+const Blog = styled('article')({
+  'background-color': 'white',
+  'box-sizing': 'border-box',
+  width: '100%',
+  display: 'flex',
+  'flex-direction': 'column',
+  'overflow-x': 'hidden',
+  'font-size': '16px',
+  position: 'relative',
+  margin: '0.5rem',
+  transition: '225ms ease-in-out',
+  outline: 'rgba(0, 0, 0, 0.15) solid 1px',
+  padding: '1em',
+  maxWidth: '400px',
+})
+
+const Title = styled('h2')({
+  'font-weight': '400',
+  margin: '0.5rem 0px',
+})
+
+const BlogDate = styled('h3')({
+  'font-size': '16px',
+  'text-transform': 'uppercase',
+  margin: '0px 0px 0.5rem',
+})
+
+const Preview = styled('p')({
+  'word-wrap': 'break-word',
+  'word-break': 'break-word',
+})
+
+const CoverPhoto = styled('img')({})
+
+const Anchor = styled(Link)({
+  '&:any-link': {
+    textDecoration: 'none',
+    color: 'hsla(0,0%,0%,0.8)',
+  },
+})
+
+export default class BlogPosts extends React.Component {
+  render() {
+    console.log(this.props)
+    const { data } = this.props
+    const sortedBlogs = data.blogPosts.edges.sort(
```

Comment:
sort with GraphQL :)

(You didn't ask for a review, but you also don't say no)